app/views/handlers/thank_you.py

```diff
@@ -1,26 +1,29 @@
 from flask import session as cookie_session
 from werkzeug.exceptions import NotFound
 
-from app.data_models.session_data import SessionData
 from app.globals import get_session_store
 from app.views.contexts.thank_you_context import (
     build_census_thank_you_context,
     build_default_thank_you_context,
 )
+from app.views.handlers.confirmation_email import ConfirmationEmail
 
 
 class ThankYou:
     DEFAULT_THANK_YOU_TEMPLATE = "thank-you"
     CENSUS_THANK_YOU_TEMPLATE = "census-thank-you"
+    PAGE_TITLE = "Thank you"
```

Comment:
This needs to be marked up for translation.
```suggestion
    PAGE_TITLE = gettext("Thank you")
```